#service to get all objects in a specific room
string room

---

cameralauncher/Object[] objects